ansible/roles/edu-sharing/tasks/edu-sharing-old/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
-  - name: Ensure gzip is present
+  - name: Ensure gzip and jq is present
```

Comment:
Why adding `jq` here? This is also included in `edusharing.yml`, isn't it? Since both tasks will be processed each run without condition, one place for `jq` should be enough

Reply:
Hello @mirjan-hoffmann 

I removed it, Yes we don't need it.